### DISCUSSION TITLE
Can not use media Query

### DISCUSSION THREAD

**Discussion by tienlx97** [facebook/stylex#574] [Q&A]
```js
const styles = stylex.create({
  leftRailContainer: {
    backgroundColor: "var(--surface-background)",
    boxSizing: "content-box",
    display: "flex",
    flexDirection: "column",
    minHeight: "inherit",
    top: 0,
    width: "360px",
    position: {
      default: "relative",
      "@media (min-width: 900px)": "fixed",
      // "@media (max-width: 899px)": stylex.firstThatWorks(
      //   "-webkit-sticky",
      //   "sticky"
      // ),
      "@media (max-width: 899px)": "sticky",
    },
    height: {
      default: null,
      "@media (max-width: 899px)": "100vh",
    },
    maxHeight: {
      default: null,
      "@media (min-width: 900px)": 0,
    },
  },
});
```

I can not use this media query in code, can you help me

**Comment by BMCwebdev** [comment]
I don’t see anything wrong here. I haven’t used firstThatWorks, not sure that’s how you’d use it, but you have that commented out anyways.

Try simplifying, just use one style with a media query and see what happens? Are you getting any type of error that may be a clue to there being a problem elsewhere?

**Reply by tienlx97** [reply]
@BMCwebdev  this is my code

https://stackblitz.com/edit/stylex-next-1r9ncs?file=app%2Fpage.tsx,app%2FNavLink.tsx,app%2Flayout.tsx

**Reply by BMCwebdev** [reply]
Wow, I forgot about this and just happened onto it today looking for a different discussion. Hey, you still working on this? I think possibly your issues was just that you wanted align-items instead of justify-content. You're styles and stylex were all working as they should.
https://stackblitz.com/edit/stylex-next-mg34rh

**Comment by nmn** [comment]
I don't see anything wrong. What result are you getting?

**Reply by tienlx97** [reply]
@nmn  This is my code

https://stackblitz.com/edit/stylex-next-1r9ncs?file=app%2Fpage.tsx,app%2FNavLink.tsx,app%2Flayout.tsx

**Reply by nmn** [reply]
Looks you're trying to apply `justifyContent` instead of `alignItems`. I saw your example, and that's the only issue I found. The styles are being generated correctly. lmk if there's something else.

Also, sorry for the late reply.
